components/header/nav/LoginMeny.js

```diff
@@ -1,25 +1,42 @@
 import MenyLink from "./MenyLink";
+import styles from "../../../styles/Nav.module.css";
+
 import { LoginContext } from "../../contexts/loginContext";
 import { useContext } from "react";
 
-export default function LoginMeny({ _class }) {
-  const {userObj} = useContext(LoginContext)
-  return (
-    <div className={_class}>
-      <MenyLink
-        _tag={"/login"}
-        _innerText={"login"}
-        _divClass={"login-meny-link"}
-      />
+export default function LoginMeny() {
+  const { userObj } = useContext(LoginContext);
 
-      {userObj.user !== null 
-      ? <MenyLink
-        _tag={"/my-page"}
-        _innerText={userObj.user}
-        _divClass={"login-meny-link"}
-      />
-    : <a></a>}
-      
+  function signOutUser() {
+    window.cookie = "_session=; expires=Thu, 01 Jan 1970 00:00:00 UTC; path=/;";
+    window.location.reload(false);
+  }
+
+  return (
+    <div className={styles.nav_utilites_containter}>
+      {userObj.user !== null ? (
+        <div className={styles.utilites_link_container}>
+          <MenyLink
+            _tag={"/my-page"}
+            _innerText={userObj.user}
+            _divClass={"login-meny-link"}
```

Comment:
The `_myprop` style (with leading `_`) is not a very common practice, and also not consistent with other parts of this codebase. It's always a good idea to agree within the team on "code style", in order to remain consistent.